tests/cephadm/test_osd_config.py

```diff
@@ -0,0 +1,55 @@
+import os
+import re
+
+from cli.cephadm.cephadm import CephAdm
+from cli.cephadm.exceptions import CephadmOpsExecutionError
```

Comment:
Add this exception to https://github.com/red-hat-storage/cephci/blob/master/cli/exceptions.py